Formula/scotch@5.rb

```diff
@@ -0,0 +1,65 @@
+class ScotchAT5 < Formula
```

Comment:
We don't want to introduce that old version, especially as it needs a patch to build. Let's stick to the current version.